thrift-logger/src/main/java/com/pinterest/singer/client/ThriftLoggerFactory.java

```diff
@@ -81,16 +80,38 @@ public void run() {
   }
 
   /**
-   * Create a default implementation of the ThriftLoggerFactory.
-   * Right now we only support the logback-based thrift logger.
-   * Call this at the start of your server to initialize the thrift logging layer.
+   * Refactoring: Let initialize() method create AuditableLogbackThriftLoggerFactory by default.
+   * AuditableLogbackThriftLoggerFactory will create AuditableLogbackThriftLogger if audit
+   * configurations are used, otherwise it will create LogbackThriftLogger. Call this at the start
+   * of your server to initialize the thrift logging layer.
    */
   public static synchronized void initialize() {
     if (THRIFT_LOGGER_FACTORY_INSTANCE != null) {
```

Comment:
the code in `initialize()` and `initializeAuditLogbackThriftLoggerFactory()` is identical.   shall we just call  `    initializeAuditLogbackThriftLoggerFactory(); ` in this method?

Reply:
initializeAuditLogbackThriftLoggerFactory() is removed  as it is duplicate code.